src/app/(profile)/profile-edit/page.tsx

```diff
@@ -1,4 +1,4 @@
-"use client";
```

Comment:
This endpoint `src/app/(profile)/profile-edit/...` does not not follow our naming conventions. It has to be something close to `.../profile/[profileId]`.